test/FIFOMarket.test.ts

```diff
@@ -12,12 +12,40 @@ import {
 const setupTestLocal = async (
```

Comment:
this function now takes an optional array of removal amounts that will be used to initialize inventory in the market contract upon setup.  there isn't net granular control over these removals such as which suppliers they belong to, what the vintages are etc. but that can be added later as we need more control over these aspects of the test cases.